src/vorta/views/repo_add_dialog.py

```diff
@@ -211,6 +214,24 @@ def password_listener(self):
             self.passwordLabel.setText(translate('utils', msg))
             return not bool(msg)
 
+    def cancel_job(self):
+        self.close()
+        try:
+            if self.job:
+                self.job.cancel()
+                logger.debug(f"Cancelled adding repo {self.values['repo_url']}")
+        except Exception as e:
+            logger.error(e)
+
+    # def cancel_queued_job(self):
+    #     self.close()
+    #     try:
+    #         if self.job or self.job.repo_id() in self.job_manager.jobs:
```

Comment:
I assume this would work 🤔

Reply:
I am sorry to tell you that it won't. But no worries. The issue that jobs aren't cancelled when being in the queue still exists for a long time now and didn't cause any problems until now. Therefore it won't hurt not fixing it in this PR. So I suggest you just ignore this potential issue.

Reply:
Okay!